components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -104,13 +104,17 @@ class Scheduler(BaseModel):
 class ResultsCache(BaseModel):
     host: str = 'localhost'
     port: int = 27017
+    db_name: str = 'clp-search'
 
     @validator('host')
     def validate_host(cls, field):
         if '' == field:
             raise ValueError(f'{RESULTS_CACHE_COMPONENT_NAME}.host cannot be empty.')
         return field
 
+    def get_uri(self):
+        return f"mongodb://{self.host}:{self.port}"
```

Comment:
```suggestion
        return f"mongodb://{self.host}:{self.port}/{self.db_name}"
```

Then we don't need to pass down (to the task) the database name as separate parameter.